### PR TITLE
Use fully qualified class names for modTemplateVar

### LINKED ISSUE #14714: MODX 3 Lack of access when working with a media source
### Summary
I created a new file source and linked it to the assets/ folder.

When I try to upload a file to TV, or open a file source, an access error appears:

source

### Environment
MODX 3

### PR DESCRIPTION
**Pull request by JoshuaLuckers**
### What does it do?
Use the fully qualified class names for modTemplateVar when possible

### Why is it needed?
To make sure we have the right class names.

### Related issue(s)/PR(s)
Fixes #14714

### CODE REVIEW

**Review by Ruslan-Aleev — APPROVED** [review]
It works, thanks :)

File: core/src/Revolution/Sources/mysql/modMediaSourceElement.php [hunk 1]
```diff
@@ -30,16 +31,16 @@ class modMediaSourceElement extends \MODX\Revolution\Sources\modMediaSourceEleme
...
             ),
-            'object_class' => 
+            'object_class' =>
             array (
                 'dbtype' => 'varchar',
                 'precision' => '100',
                 'phptype' => 'string',
                 'null' => false,
-                'default' => 'modTemplateVar',
+                'default' => 'MODX\\Revolution\\modTemplateVar',
```
Comment (opengeek) [inline core/src/Revolution/Sources/mysql/modMediaSourceElement.php:40]: This change needs to be made in the schema files and updated by using the xpdo parse-schema command.

**Review by opengeek — CHANGES_REQUESTED** [review]

File: core/src/Revolution/Sources/mysql/modMediaSourceElement.php (hunk 1 above)
Comment (JoshuaLuckers) [reply core/src/Revolution/Sources/mysql/modMediaSourceElement.php:40]: @opengeek im not sure what to change in the schema file to reflect this change, maybe I’m overlooking something.

Comment (Mark-H) [reply core/src/Revolution/Sources/mysql/modMediaSourceElement.php:40]: https://github.com/modxcms/revolution/blob/3.x/core/model/schema/modx.sources.mysql.schema.xml#L69

Comment (opengeek) [reply core/src/Revolution/Sources/mysql/modMediaSourceElement.php:40]: I'll get this resolved today and post the command so we have it for future reference when updating the model.

Comment (opengeek) [reply core/src/Revolution/Sources/mysql/modMediaSourceElement.php:40]: Here were the commands to update the files once the schemas were updated:

```
core/vendor/bin/xpdo parse-schema --config=core/model/schema/config.php --update=1 --psr4=MODX\\ mysql core/model/schema/modx.sources.mysql.schema.xml core/src/
core/vendor/bin/xpdo parse-schema --config=core/model/schema/config.php --update=1 --psr4=MODX\\ sqlsrv core/model/schema/modx.sources.sqlsrv.schema.xml core/src/
```

Let me know if you have any questions.

Comment (JoshuaLuckers) [reply core/src/Revolution/Sources/mysql/modMediaSourceElement.php:40]: Done, thanks @Mark-H and @opengeek

**Pull request merged**